src/S_Polyhedron/Polyhedron_algorithms.cpp

```diff
@@ -71,6 +71,44 @@ const Polyhedron_algorithms& Polyhedron_algorithms::operator =(const Polyhedron_
 }
 
 
+int Polyhedron_algorithms::getEdgeKey(PolyhedronEdge e)
+{
+  return ((e.a<e.b)?(e.a*vertexes_.size()+e.b):(e.b*vertexes_.size()+e.a));
+}
+
+void Polyhedron_algorithms::getEdges()
+{
+  PolyhedronEdge edge;
+  std::vector<PolyhedronEdge> triangleEdges;
+  std::set<int, std::greater<int>> edgesSet;
+  for(size_t i = 0; i < triangles_.size(); i++)
+  {
+    edge.a = triangles_[i].a;
+    edge.b = triangles_[i].b;
+    edge.computeEdge(vertexes_);
+    triangleEdges.push_back(edge);
+    
+    edge.a = triangles_[i].b;
+    edge.b = triangles_[i].c;
+    edge.computeEdge(vertexes_);
+    triangleEdges.push_back(edge);
+    
+    edge.a = triangles_[i].c;
+    edge.b = triangles_[i].a;
+    edge.computeEdge(vertexes_);
+    triangleEdges.push_back(edge);
+
+    for(auto j = triangleEdges.begin(); j != triangleEdges.end(); j++)
+    {
+      if(!edgesSet.count(getEdgeKey(*j)))
+      {
+        edges_.push_back(*j);
```

Comment:
`edges_` is not cleared first. Is it intended?

Reply:
You are right and it should be cleared also in the clear() method

Fixed.